### PR TITLE
Nomenclature

### PR DESCRIPTION
**Pull request by jeremyphilemon**
Opening a PR for discussion.
Fixes #70.

### CODE REVIEW

**Comment by digi0ps** [comment]
I am working on the voting bug. I will join soon!

**Comment by jeremyphilemon** [comment]
@digi0ps Feast your eyes xD

File: RestApi/views.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
-from django.shortcuts import render
+from django.shortcuts import render  #  unused
```
Comment (jeremyphilemon) [inline RestApi/views.py]: These imports are unused

Comment (digi0ps) [inline RestApi/views.py]: Sure!

**Pull request merged**